server/controllers/accesses.js

```diff
@@ -76,24 +76,32 @@ export async function create(req, res) {
 // Fetch operations using the backend. Note: client needs to get the operations
 // back.
 export async function fetchOperations(req, res) {
+    let access = req.preloaded.access;
     try {
         // Fetch operations
-        let access = req.preloaded.access;
         await commonAccountManager.retrieveOperationsByAccess(access);
         res.sendStatus(200);
     } catch (err) {
+        if (err.errCode) {
+            access.fetchStatus = err.errCode;
+            await access.save();
```

Comment:
Can this be put in retrieveOperationsByAccess directly? So that it doesn't leak in the controllers.